src/corelibs/U2Core/src/globals/U2SafePoints.h

```diff
@@ -75,7 +87,7 @@
 */
 #define SAFE_POINT_EXT(condition, extraOp, result) \
     if (Q_UNLIKELY(!(condition))) { \
-        assert(condition); \
+        U2::U2SafePoints::fail(); \
```

Comment:
I suggest to add the following message:
`QString("Trying to recover from the unknown error at %1:%2").arg(__FILE__).arg(__LINE__))`
It would be better than an empty line

Reply:
Good idea. We can actually print the original condition here too. Fixed